api/job/v1beta1/job.proto

```diff
@@ -69,11 +72,26 @@ message GetAvailableJobsRequest {
   int32 limit = 4;
 }
 
-message AvailableJob {
-  JobInfo job = 1;
-  int32 distance_meters = 2;
+message Location {
+  double lat = 1;
+  double lng = 2;
```

Comment:
We should have a [LatLng](https://github.com/googleapis/googleapis/blob/master/google/type/latlng.proto) type if we don't already